woocommerce/woocommerce
# Stop caching country calling codes from i18n/phone.php

### Changes proposed in this Pull Request:

The country calling codes used by `WC_Countries::get_country_calling_code()` is a hardcoded PHP array.  Setting and retrieving it from cache only adds overhead.  So the cache usage here should be removed.

This was initially suggested in the code review of #22693, but must have been ignored.

Note: This is a minor change with likely very little real impact, however it was the first issue I came across when starting to explore the way we're using cache within WooCommerce core.



### How to test the changes in this Pull Request:

I'm not sure how to test this.  I tried the following based on the instructions from #22693, however, it appears as though the latest PayPal integration doesn't follow the same flow and no longer fills in the address information from the WooCommerce order.  

1. Setup PayPal sandbox on the test site.
2. Under PayPal settings:
 - Set landing page to Non-PayPal Account
 - Set "Send checkout billing data to card fields" to "Use WC checkout form data"
3. In an incognito window, attempt to checkout using PayPal.

This doesn't appear to work with our without this change.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message Stop caching country calling codes from i18n/phone.php

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
